modules/ecs-ec2/otel_config_metrics.tftpl.yaml

```diff
@@ -83,43 +76,42 @@ extensions:
 service:
   extensions:
     - health_check
-
+  telemetry:
+    metrics:
+      address: 0.0.0.0:8888
+      level: detailed
   pipelines:
     logs:
       receivers:
+        - otlp
```

Comment:
why do we need to add it?

Reply:
because:
1. probably nonzero customer count where infosec policy is no console logging to avoid `docker logs` exposure. E.g. regulatory data controls.
2. standardize on OTEL logging, or are migrating towards OTEL from competitor. Why should we be the blocker if customer is ready to move, now, or tomorrow?
3. super low-cost and low-risk.
4. Later we may want to expand this TF module to add optional OTEL Agent Gateway deployment pattern, and 2-tier pattern. Maybe. This will reuire OTLP enabled, I believe.
5. It is signalling that we as a company are in support of OTLP. Do we need an internal discussion and agreement to greenlight the decision officially, or just proceed?
6. Have an all-in-one OTEL-enabled daemon endpoint ready-to-go with low-friction on every EC2 instance. 
7. Improve customer experience. Customers requiring OTEL do not need to provide a customize our OTEL config yaml 🤦🏻‍♂️

Reasons against:
1. Inconsistent with CloudFront flavor: does not have otlp for logs pipeline.
2. Customers may fall into anti-pattern of enabling both console logs and OTLP, resulting in double-cost.

Reply:
I dont see a reason to add this in our default, in ECS default integration this is not needed
if there will be a need for it customer can update their config and easily add it